### DISCUSSION TITLE
有知道MidJourney插件怎么使用的吗?完全搞不懂怎么使用

### DISCUSSION THREAD

**Discussion by wangshipei** [lobehub/lobehub#1139] [General]
有知道MidJourney插件怎么使用的吗?完全搞不懂怎么使用

**Comment by arvinxx** [comment]
如果没有部署过 midjourney-proxy 的话，应该是无法正常使用的。

**Reply by wangshipei** [reply]
部署过了，在midjourney-webui.lobehub.com上面可以用，但是在lobechat聊天界面调用后，无法生成图片

**Reply by arvinxx** [reply]
> 部署过了，在midjourney-webui.lobehub.com上面可以用，但是在lobechat聊天界面调用后，无法生成图片

可能是提示词的问题？你看下是不是有 `--style xxx` 这样的描述？可能这个 style 不是mj支持的

**Comment by wangshipei** [comment]
![image](https://github.com/lobehub/lobe-chat/assets/54164572/b4955f11-2ec6-4b86-bbd4-31b01cfacd82)
直接复制catjourney的提示词也是不行

**Reply by wangshipei** [reply]
点击那个画笔，也是没办法生成

**Reply by arvinxx** [reply]
你按下 F12，然后点下画笔后看下请求是啥样的

**Reply by wangshipei** [reply]
![c0b0e5458d003d15b9879deb6f394709](https://github.com/lobehub/lobe-chat/assets/54164572/c3164641-e8fd-42dc-8672-9704eb76aada)
![ff1425b4bbdf753860cfdd059597483c](https://github.com/lobehub/lobe-chat/assets/54164572/81ef6730-e88c-4814-9280-15c756f5b1bc)

**Reply by fxywolf** [reply]
我和你同样的问题，我是部署在本地自己电脑上使用。

**Reply by arvinxx** [reply]
我最近看下，可能有 bug

**Comment by wangshipei** [comment]
![image](https://github.com/lobehub/lobe-chat/assets/54164572/91dede64-079a-4a5f-81e2-03e155a24fea)
在网页上没问题

**Comment by wangshipei** [comment]
我是部署在 vercel 上，有设置搜索引擎的 serpapi 的 key，会跟这个有关系吗？

**Reply by wangshipei** [reply]
然后 domains 有设置自己的域名
